src/qt/recentrequeststablemodel.h

```diff
@@ -25,10 +27,10 @@ class RecentRequestEntry
     SendCoinsRecipient recipient;
 
     SERIALIZE_METHODS(RecentRequestEntry, obj) {
-        unsigned int date_timet;
-        SER_WRITE(obj, date_timet = obj.date.toTime_t());
+        qint64 date_timet;
```

Comment:
I don't think this can be changed without breaking wallet.dat

Reply:
I'm going to revert this. We can leave it as unrelated.